dvc/scm/git.py

```diff
@@ -183,13 +183,12 @@ def _get_gitignore(self, path):
         return entry, gitignore
 
     def _ignored(self, path):
-        from git.exc import GitCommandError
+        from dulwich import ignore
+        from dulwich.repo import Repo
 
-        try:
-            self.repo.git.check_ignore(path)
-            return True
-        except GitCommandError:
-            return False
+        repo = Repo(self.root_dir)
```

Comment:
Wasteful to have a repo instance created each time here, but doing so for drop-in simplicity. Likely dulwich will replace everything except `Git.clone()` in the near future and so `Repo` will move to `__init__`.